components/Pages/Project/AirdropGitcoinSupporters.tsx

```diff
@@ -0,0 +1,446 @@
+                                <div>{metadata.description}</div>
+                            </div>
+                        </div>
+                    </div>
+                )}
+                <button
+                    onClick={handleMintNFTs}
+                    disabled={!imageIPFSHash || isMinting || isSuccess}
+                    className="px-4 py-2 bg-blue-600 text-white rounded-md hover:bg-blue-700 focus:outline-none focus:ring-2 focus:ring-blue-500 focus:ring-offset-2 disabled:bg-gray-400 disabled:cursor-not-allowed mb-2"
+                >
+                    {isMinting ? "Minting..." : isSuccess ? "NFTs minted successfully!" : "Mint NFTs to Contributors"}
+                </button>
+                {isSuccess && <p className="text-green-500 text-sm">
+                    View transaction on <TransactionLink chainId={chainId} transactionHash={txData} />
+                </p>}
+                {mintError && <p className="text-red-500 text-sm">
+                    {mintError.message.includes("insufficient funds") ? "Insufficient funds for transaction" : mintError.message.includes("Project already exists") ? "Project already minted" : mintError.message}
+                </p>}
+                <div className="text-sm text-gray-600 dark:text-gray-300">
+                    <PlatformFeeNote />
+                </div>
+            </div>
+            <div className="w-1/2 h-full ">
+                {fileUploading ? (
+                    <div className="w-full min-h-80   flex items-center justify-center bg-gray-200 dark:bg-gray-700 rounded-xl">
+                        <div className="animate-spin rounded-full h-16 w-16 border-t-2 border-b-2 border-blue-500">
+                        </div>
+                    </div>
+                ) : imageIPFSHash ? (
+                    <img
+                        src={`/api/img-proxy?url=https://gateway.pinata.cloud/ipfs/${imageIPFSHash}`}
+                        alt="NFT Image"
+                        className="w-full h-auto rounded-xl bg-zinc-300"
+                    />
+                ) : (
+                    <div className="w-full  min-h-80   flex items-center justify-center bg-gray-200 dark:bg-gray-700 rounded-xl">
+                        <p className="text-gray-500 dark:text-gray-400">No image uploaded</p>
+                    </div>
+                )}
+            </div>
+        </div>
+    );
+}
+
+
+
+
+
+export const GitcoinAirdropsManager = () => {
+    const { address, isConnected } = useAccount();
+    const { chain } = useAccount();
+
+    const [projectURL, setProjectURL] = useState("")
+    const [loading, setLoading] = useState(false)
+    const [page, setPage] = useState(1)
+    const [itemsPerPage, setItemsPerPage] = useState(10)
+    const [totalPages, setTotalPages] = useState(0)
+    const [projectData, setProjectData] = useState<{ details: any, donations: any }>(
+        {
+            details: null,
+            donations: []
+        }
+    )
```

Comment:
I think we should have some types there, any can be very tricky. (Type at least what you are using on the constant)